In [2]:
%matplotlib inline
from pylab import *

![image](coilCrossSection.png) 


$\alpha \in \left[-\frac{a}{2},+\frac{a}{2}\right]$

$\beta \in \left[-\frac{b}{2},+\frac{b}{2}\right]$

$a = nd$

$b = md,$

where d is the diameter of a wire, assuming that the insualtion thickness is negligible.  Since d is used for the copper conductance calculation in the next section, the accurate version of these equations is 

$a = n(d + 2t)$ and $b = m(d + 2t),$

where t is the insulation thickness.  In most cases I will ignore that and use the above approximation that $\frac{t}{d} \ll 1$


number of turns $ = N = mn$

length = 

$$\ell = m\sum_{\alpha = -\frac{a}{2} }^{\frac{a}{2}} 2\pi(R + \alpha)$$

From https://en.wikipedia.org/wiki/American_wire_gauge :

$n = -39\log_{92} \left( \frac{d_{n}}{0.005~\mathrm{inch}} \right)+36 = -39\log_{92} \left( \frac{d_{n}}{0.127~\mathrm{mm}} \right)+36$


$d_n = 0.005~\mathrm{inch} \times 92 ^ \frac{36-n}{39} = 0.127~\mathrm{mm} \times 92 ^ \frac{36-n}{39}$


Conductivity of wires is compared to the International Annealed Copper Standard (IACS), which is 5.8e7 Siemens/meter or 1.72 micro-Ohm cm, and this is for standard annealed copper wire at 20 degrees C.  

$R = \rho\ell/A$, 

where I now have all distances in cm.  And I've introduced redundant notation here with R, but it should be clear from context.  Or I'll change the other one, the big radius, because resistance must always be capital R. 

Now suppose that the length is 500 cm.  


These numbers need to be checked and re-checked, and graphs made with graphical "calculators".  Also mass and volume should be calculated, and optimization rules of thumb developed.  Also the number of turns. 

$\Omega/m \left(\frac{1 m}{100 cm}\right)\left(\frac{2.54 cm}{1 in}\right)\left(\frac{12 in}{1 ft}\right)$ 

In [3]:
print("AWG"), 
print("            | "),
print("Diameter [mm]"),
print(" | "),
print("Diameter [mils]"),
print(" | "),
print("Area [mm^2]"),
print(" | "),
print("R [mOhms/m]"),
print(" | "),
print("R [mOhms/ft]")

print("-----------"), 
print(" | "),
print("-----------"), 
print(" | "),
print("-----------"), 
print(" | "),
print("-----------"), 
print(" | "),
print("-----------")



for AWG in range(37):
    print AWG, 
    print("          | "),
    d_mm = round(0.127*92.0**((36.0-AWG)/39.0),2)
    print d_mm,
    d_mils = round(5.0*92.0**((36.0-AWG)/39.0),1)
    print("      | "),
    print d_mils,
    print("      | "),
    A = pi*(d_mm/2)**2
    A = round(A,3)
    print A,
    A_cm = A/100 # convert from mm^2 to cm^2
    rho = 1.72e-6
    R_m = 1000*100*rho/A_cm  #R per unit meter
    R_m = round(R_m,1)
    print("|"),
    print R_m,
    print("|"),
    R_ft = R_m*2.54*12/100
    R_ft = round(R_ft,1)
    print R_ft

AWG             |  Diameter [mm]  |  Diameter [mils]  |  Area [mm^2]  |  R [mOhms/m]  |  R [mOhms/ft]
-----------  |  -----------  |  -----------  |  -----------  |  -----------
0           |  8.25       |  324.9       |  53.456 | 0.3 | 0.1
1           |  7.35       |  289.3       |  42.429 | 0.4 | 0.1
2           |  6.54       |  257.6       |  33.593 | 0.5 | 0.2
3           |  5.83       |  229.4       |  26.695 | 0.6 | 0.2
4           |  5.19       |  204.3       |  21.156 | 0.8 | 0.2
5           |  4.62       |  181.9       |  16.764 | 1.0 | 0.3
6           |  4.12       |  162.0       |  13.332 | 1.3 | 0.4
7           |  3.66       |  144.3       |  10.521 | 1.6 | 0.5
8           |  3.26       |  128.5       |  8.347 | 2.1 | 0.6
9           |  2.91       |  114.4       |  6.651 | 2.6 | 0.8
10           |  2.59       |  101.9       |  5.269 | 3.3 | 1.0
11           |  2.3       |  90.7       |  4.155 | 4.1 | 1.2
12           |  2.05       |  80.8       |  3.301 | 5.2 | 1.6
13      

In [8]:
l = 100*5/.168
d = 3
c = pi*d
print l
n = l/c
print n

2976.19047619
315.783617246


27 AWG is 0.168 mOhm per meter, so for 5 ohms it is 2976 cm.  If diameter is 3 cm, circumpherence is pi times diameter, then divide that into the total lenght to get number of turns, which is 320 or so.  I probably wound 500 turns on my first attempt, hence the 8 ohms.  

Also for 320 turns the square root of that is

In [9]:
sqrt(320)

17.888543819998318

18 on a side, times a wire diameter of 0.36 mm so that's 

In [10]:
18*0.36

6.4799999999999995

about 6 and a half mm, with a 30 mm larger diameter for the "torus". More care must be taken with bottle cap dimensions.  This means the inner and outer main diameters are 36.5 and 23.5 mm.  